package/environment.js

```diff
@@ -25,6 +26,8 @@ const getLoaderList = () => {
 const getPluginList = () => {
   const result = new ConfigList()
   result.append('Environment', new webpack.EnvironmentPlugin(JSON.parse(JSON.stringify(process.env))))
+  result.append('NamedModules', new webpack.NamedModulesPlugin())
```

Comment:
> This plugin will cause the relative path of the module to be displayed when HMR is enabled. Suggested for use in development. (https://webpack.js.org/plugins/named-modules-plugin/)

Should we be using this in all environments?

Reply:
Good spot 👁 , will fix 👍